### DISCUSSION TITLE
Github token permissions for workflow seem wrong

### DISCUSSION THREAD

**Discussion by PaulRudin** [community/community#38536] [Actions]
I have an workflow yaml with two jobs. The second job includes a permissions block:
```yaml
permissions:
    contents: 'write'
    id-token: 'write'
```
But when I look at  the "set up job" output from a run of the job I see:
```
GITHUB_TOKEN Permissions
  Contents: read
  Metadata: read
  ```

and one of the steps for the job fails with `Error: Error 403: Resource not accessible by integration`. That step should just need the `contents: write` permission as far as I know.

Why does the setup job output show something different from what's granted in the permission block for the job?

**Comment by airtower-luna** [comment]
Is that a job triggered by a `pull_request` event? If that event comes from a fork its permissions are limited to read-only.

**Comment by ossanna16** [comment]
Hi there @PaulRudin and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
